src/lib/djerba/plugins/tar/swgs/render.py

```diff
@@ -0,0 +1,132 @@
+"""
+This script describes a list of functions to render the sWGS data from the json into HTML.
+AUTHOR: Aqsa Alam
+"""
+
+# IMPORTS
+import re
+from markdown import markdown
+from time import strftime
+from string import Template
+import djerba.plugins.tar.swgs.constants as constants
+
+class html_builder:
+  
+  TR_START = '<tr style="text-align:left;">'
+  TR_END = '</tr>'
+    
+    
+  # ------------------- TABLE FORMAT FUNCTIONS -------------------
+  
+  def section_cells_begin(self, section_title, main_or_supp):
+    """
+    Describes how the section cells begin.
+    Taken from the original json_to_html.py from the non-plugin Djerba.
+    
+    Begin a cell structure with title in left-hand cell, body in right-hand cell
+    """
+    permitted = ['main', 'supp']
+    if main_or_supp not in permitted:
+        msg = "Section type argument '{0}' not in {1}".format(main_or_supp, permitted)
+        self.logger.error(msg)
+        raise RuntimeError(msg)
+    template = '<hr class="big-white-line" ><div class="twocell{0}"><div class="oneoftwocell{0}">{1}</div><div class="twooftwocell{0}" ><hr class="big-line" >'
+    cell = template.format(main_or_supp,section_title)
+    return cell
+  
+  def section_cells_end(self):
```

Comment:
this function is in a lot of places

Reply:
Agreed. I've put equivalent functions in `util` -- if necessary I can do a small pull request to merge those to dev, to make them available for the TAR plugin